lms/djangoapps/course_api/tests/test_views.py

```diff
@@ -85,8 +85,13 @@ def test_as_inactive_user(self):
         self.verify_response(params={'username': inactive_user.username})
 
     def test_missing_username(self):
+        response_to_anonymous = self.verify_response(expected_status_code=200)
```

Comment:
Ah, I see what you mean now. I can shorten this test to just test that a missing username call gets a 200 with data, and try to write another test that deals with a case where an anonymous and signed-in user would get different responses.